sdk/eventhub/event-processor-host/src/modelTypes.ts

```diff
@@ -191,4 +191,9 @@ export interface EventProcessorHostOptions extends FromConnectionStringOptions {
    * provides the token for authentication. Default value: **`SasTokenProvider`**.
    */
   tokenProvider?: TokenProvider;
+  /**
+   * @property {string} [userAgent] The user agent that needs to be appended to the built in
+   * user agent string.
+   */
+  userAgent?: string;
```

Comment:
EventProcessorHostOptions extends ClientOptionsBase which has this property. Why are we adding this here?

Reply:
Good catch! I didn't check the base types.

On a side note, If `FromTokenProviderOptions` derives from `ClientOptions` we can remove the `tokenProvider` field too.

Reply:
> If FromTokenProviderOptions derives from ClientOptions

NVM. It turns out that at some point of time it was deriving from ClientOptions but later changed.